doc/src/manual/constructors.md

```diff
@@ -554,8 +554,8 @@ julia> type SummedArray{T<:Number,S<:Number}
            sum::S
        end
 
-julia> SummedArray(Int32[1; 2; 3], Int8(6))
-SummedArray{Int32,Int8}(Int32[1,2,3],6)
+julia> SummedArray(Int32[1; 2; 3], Int128(6))
+SummedArray{Int32,Int128}(Int32[1,2,3],6)
```

Comment:
maybe have this one give the `Int32` type that the following paragraph talks about preventing, then try to construct it in that block and show the error?